### PR TITLE
Filterupdate

### PR DESCRIPTION
**Pull request by danmcgill6**
### Assignee Tasks

- [ ] added unit tests (or none needed)
- [ ] written relevant docs (or none needed)
- [ ] referenced any relevant issues (or none exist)

### Guidelines

Please add a description of this Pull Request's motivation, scope, outstanding issues or potential alternatives, reasoning behind the current solution, and any other relevant information for posterity.

---

*Your PR Notes Here*

### CODE REVIEW

File: client/components/common/Filter.js [hunk 1]
```diff
@@ -6,7 +6,9 @@ class Filter extends Component {
       super(props)
     }
 
-    cheapFilter() {
+    cheapFilter(event) {
+      console.log("HEY")
+      console.log(event.target.value)
```
Comment (bruceleong) [inline client/components/common/Filter.js]: delete console.log

File: client/components/pages/AllProducts.js [hunk 2]
```diff
@@ -8,30 +8,52 @@ import { addToCart } from '../../store/cart'
...
     this.props.fetchData();
   }
 
-  // onClicking(product) {
-  //   console.log(product)
-  //   // dispatch(fetchOneProduct(product)
-  // }
+  checkboxClicked(event) {
+
+    console.log('click')
```
Comment (bruceleong) [inline client/components/pages/AllProducts.js]: delete console.log

Comment (bruceleong) [reply client/components/pages/AllProducts.js]: see line 39

**Review by bruceleong — CHANGES_REQUESTED** [review]

**Review by mfriedland — APPROVED** [review]
looks beautiful now

**Review by bruceleong — APPROVED** [review]
looks good, thank you for making changes - console.log, filter updates.

**Pull request merged**